README.md

```diff
@@ -173,6 +173,38 @@ console.log(await promptWithCalculator("What is 2 + 2?"));
 
 We'll likely explore more specific APIs for tool- and function-calling in the future; follow along in [issue #7](https://github.com/webmachinelearning/prompt-api/issues/7).
 
+### Structured output or JSON output
+
+To help with programmatic processing of language model responses, the prompt API supports structured outputs defined by a JSON schema.
+
+```js
+const session = await ai.languageModel.create();
+
+const responseJSONSchemaObj = new AILanguageModelResponseSchema({
+  "type" : "object",
+  "required" : ["Rating"],
+  "additionalProperties" : false,
+  "properties" : {
+    "Rating": {
+      "type": "number"
+      "minimum": 0,
+      "maximum": 5
+    }
+  }
+});
```

Comment:
```suggestion
const responseJSONSchemaObj = new AILanguageModelResponseSchema({
  type: "object",
  required: ["Rating"],
  additionalProperties: false,
  properties: {
    Rating: {
      type: "number",
      minimum: 0,
      maximum: 5,
    },
  },
});
```

(comma after `"number"` was missing, and the spacing around `:` was inconsistent. Removing quotes on property names is just a style choice.)